app/controllers/user.js

```diff
@@ -0,0 +1,19 @@
+const userService = require('../services/user');
+const logger = require('../logger');
+const errors = require('../errors');
+
+exports.createUser = async (req, res, next) => {
+  try {
+    const { body } = req;
+    const user = await userService.createUser(body);
+    if (user) {
+      logger.info(`User ${user.firstName} was created successfull`);
+      return res.status(200).send({ user });
+    }
+    const errorMessage = `User ${user.firstName}  couldn't be created`;
+    logger.info(errorMessage);
+    return next(errors.databaseError(errorMessage));
```

Comment:
This error handling must be done from the service, here only that error is received and it is passed to the `next` function.